icons/rubymine/rubymine-plain-wordmark.svg

```diff
@@ -1 +1,3 @@
988 2.426 2.445h-4.914c.258-1.48 1.113-2.445 2.488-2.445zm-57.883.148c1.528 0 2.813 1.27 2.813 3.18v.043c0 1.934-1.289 3.176-2.813 3.176-1.523 0-2.793-1.266-2.793-3.176v-.043c0-1.922 1.266-3.18 2.793-3.18zm-51.05 6.106h5.648v.941H7.523v-.941zm18.867 8.46v5.68h.601v-.769c.33.531.914.856 1.54.856.971 0 1.921-.778 1.921-2.094v-.016c0-1.332-.958-2.086-1.922-2.086a1.8 1.8 0 0 0-1.539.899v-2.47h-.601zm36.632.118v.664h.684v-.664h-.684zm-22.496.117v3.653c0 .863-.45 1.308-1.066 1.308-.578 0-.918-.277-1.254-.777l-.441.379a1.86 1.86 0 0 0 1.68.96 1.65 1.65 0 0 0 1.199-.452 1.95 1.95 0 0 0 .503-1.43v-3.64h-.62zm9.5 0v5.438h2.422c1.192 0 1.977-.55 1.977-1.485v-.015c0-.75-.524-1.12-1.168-1.317a1.317 1.317 0 0 0 .863-1.254v-.015c.008-.324-.12-.64-.348-.871a2 2 0 0 0-1.425-.48h-2.32zm-2.859.207v1.215h-.559v.531h.559v2.426c0 .817.492 1.137 1.137 1.137a1.494 1.494 0 0 0 .722-.172v-.5a1.25 1.25 0 0 1-.582.14.587.587 0 0 1-.504-.167.59.59 0 0 1-.171-.504v-2.36h1.273v-.53H47.77V71.82h-.602zm3.469.344h1.656c.781 0 1.207.347 1.207.887v.015c0 .641-.517.973-1.254.973h-1.61v-1.875zm-6.453.785c-1.118 0-1.93.934-1.93 2.094v.016a2.014 2.014 0 0 0 .555 1.476 2.007 2.007 0 0 0 1.453.617 2.09 2.09 0 0 0 1.656-.754l-.375-.335a1.66 1.66 0 0 1-1.266.57 1.41 1.41 0 0 1-1.418-1.363h3.2a1.59 1.59 0 0 0 0-.196c0-1.175-.688-2.125-1.875-2.125zm22.75 0a1.534 1.534 0 0 0-1.38.785v-.699h-.597v4.031h.598v-2.304c-.02-.327.102-.653.332-.89a1.174 1.174 0 0 1 .875-.364c.71 0 1.113.46 1.113 1.215v2.347h.602v-2.507c0-.961-.571-1.614-1.543-1.614zm3.972.004c-.793 0-1.37.457-1.37 1.152v.016c0 .738.702.957 1.331 1.137.528.148 1.004.3 1.004.707v.015c0 .386-.348.641-.832.641a2.373 2.373 0 0 1-1.394-.504l-.305.426a2.754 2.754 0 0 0 1.668.59c.8 0 1.433-.453 1.433-1.207v-.024c0-.722-.67-.964-1.293-1.152-.535-.164-1.035-.312-1.035-.691v-.016c0-.332.3-.582.778-.582a2.272 2.272 0 0 1 1.191.387l.27-.45a2.714 2.714 0 0 0-1.446-.445zm-13.101.012a1.734 1.734 0 0 0-1.645 1.12v-1.05h-.597v4.024h.597v-1.606c0-1.21.719-1.84 1.594-1.84h.05v-.648zm2.474 0c-.059-.002-.118 0-.177.004a3.254 3.254 0 0 0-1.461.336l.18.5a2.884 2.884 0 0 1 1.218-.297c.758 0 1.2.374 1.2 1.097v.141A4.235 4.235 0 0 0 60 74.578c-1.016 0-1.719.449-1.719 1.297v.016c0 .828.754 1.254 1.504 1.242a1.762 1.762 0 0 0 1.453-.695v.609h.575v-2.461a1.6 1.6 0 0 0-.41-1.176 1.765 1.765 0 0 0-1.124-.445zm-29.424.055 1.829 4.015c-.243.567-.466.754-.825.754a1.217 1.217 0 0 1-.629-.148l-.203.472a1.805 1.805 0 0 0 .856.203c.605 0 .996-.3 1.34-1.136l1.687-4.145h-.64l-1.29 3.34-1.464-3.355h-.66zm32.207.015v4.016h.598v-4.016h-.597zm-18.894.422c.801 0 1.22.637 1.281 1.367h-2.59c.08-.785.606-1.367 1.309-1.367zm-15.746.031c.762 0 1.418.605 1.418 1.555v.016c0 .972-.641 1.546-1.418 1.546a1.506 1.506 0 0 1-1.457-1.554v-.016a1.507 1.507 0 0 1 1.457-1.547zm22.215 1.094h1.718c.95 0 1.446.34 1.438.934v.015c0 .61-.516.977-1.34.977h-1.816v-1.926zm9.422.438a4.164 4.164 0 0 1 1.18.171v.391c0 .636-.598 1.098-1.356 1.086-.535 0-1.004-.297-1.004-.8v-.013c0-.507.427-.835 1.18-.835z"/>
```

Comment:
Missing fill?
Any reason for having this icon black while the other plain icon is orange?

Reply:
Related to the #1291 thread